src/content/en/2022/performance.md

```diff
@@ -1,22 +1,919 @@
+
+Pages in this chart are more densely packed in the bottom left corner, which signifies the higher degree of correlation between FID and TBT. There's still a smear, but it's not as pronounced.
+
+The Kendall and Spearman coefficients of correlation for this distribution are 0.34 and 0.45, respectively.
+
+<figure>
+  <blockquote>
+  <p>First, is INP correlated with TBT? Is it more correlated with TBT than FID? Yes and yes!</p>
+  <p>But they are both correlated with TBT; is INP catching more problems with main thread blocking JavaScript? We can break down the percent of sites meeting the "good" threshold: yes it is!</p>
+  </blockquote>
+  <figcaption>—Annie Sullivan on <a href="https://twitter.com/anniesullie/status/1525161893450727425">Twitter</a></figcaption>
+</figure>
+
+As Annie notes, both metrics are correlated with TBT, but she concludes that INP is more strongly correlated, making it a better responsiveness metric.
+
+### FID conclusions
+
+These results show that sites absolutely do have responsiveness issues, despite the rosy picture painted by FID. Regardless of whether INP becomes a CWV metric, your users will thank you if you start optimizing it now.
+
+Nearly one in ten mobile sites are leaving free performance on the table by failing to disable double-tap to zoom. This is something all sites should be doing; it's only one line of HTML and it benefits both FID and INP. Run Lighthouse on your page and look for the <a hreflang="en" href="https://web.dev/viewport/">viewport</a> audit to be sure.
+
+By taking a hypothetical look at INP as a CWV, we can see just how much work there is to be done just to get back to FID-like levels. The most popular mobile websites would be especially affected by such a change as a consequence of their (over)use of JavaScript. Some CMSs and JavaScript frameworks would be hit harder than others, and it'll take an ecosystem-wide effort to collectively rein in the amount of client-side work that they do.
+
+## Conclusion
+
+As the industry continues to learn more about CWV, we're seeing steady improvement both in terms of implementation and across all top-level metric values themselves. The most visible performance optimization strides are at the platform level, like Android and bfcache improvements, given that their impact can be felt across many sites at once. But let's look at the most elusive piece of the performance puzzle: individual site owners.
+
+Google's decision to make CWV part of search ranking catapulted performance to the top of many companies' roadmaps, especially in the [SEO](./seo#core-web-vitals-cwv) industry. Individual site owners are certainly working hard to improve their performance and played a major role in the CWV improvements over the last year, even if those individual efforts are much harder to spot at this scale.
```

Comment:
Is "played a major role" a bit strong given we can't back that up and have stated it was other changes outside of this that likely had the biggest impact?

WDYT about this:

```suggestion
Google's decision to make CWV part of search ranking catapulted performance to the top of many companies' roadmaps, especially in the [SEO](./seo#core-web-vitals-cwv) industry. Individual site owners are certainly working hard to improve their performance and likely played a major role in the CWV improvements over the last year, even if those individual efforts are much harder to spot at this scale.
```

Reply:
Apparently discuss and agreed upon so we'll go with this!